### PR TITLE
Added updated samples of Open Tech Summit 2017 and NextCloud 2017

### PR DESCRIPTION
**Pull request by Princu7**
Partially fixes the issue #205

### CODE REVIEW

**Comment by Princu7** [comment]
Added the updated samples of Open Tech Summit 2017 and Nextcloud Conference 2017 with the new attributes. The event can be tested on this test server which contains the code for the new JSON model.
http://peaceful-earth-19050.herokuapp.com/

The PR for this test server is open on the webapp and after the events have been updated, it can be merged and then the auto deployment of the events will start working again.
fossasia/open-event-webapp#1448

* Open Tech Summit 2017 endpoint
https://raw.githubusercontent.com/Princu7/open-event/a2673b636fefb78b92f852e8a2ef0cd053ffc856/sample/OpenTechSummit2017/

* Nextcloud conference github endpoint
https://raw.githubusercontent.com/Princu7/open-event/a2673b636fefb78b92f852e8a2ef0cd053ffc856/sample/NextcloudConference2017

@mariobehling @niranjan94 @aayusharora Please review. Thanks :)

File: sample/NextcloudConference2017/event [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+        {
+            "link": "https://www.facebook.com/Nextclouders/",
+            "id": 127,
+            "name": "Facebook"
+        }
+    ],
+    "longitude": 13.32689,
+    "email": null,
+    "large-image-url": "https://storage.googleapis.com/eventyay.com/events/181/large/eDk5ZEFSWU/fe3ec0c5-26e3-46d0-9407-e2d50353862c.jpg",
+    "call-for-papers": {
```
Comment (Shailesh351) [inline sample/NextcloudConference2017/event]: Here it is `speakers-call`

File: sample/NextcloudConference2017/event [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+        "start-date": "2017-03-15T16:00:00+00:00",
```
Comment (Shailesh351) [inline sample/NextcloudConference2017/event]: `starts-at`, `ends-at`,

File: sample/NextcloudConference2017/event [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+        "end-date": "2017-08-24T17:00:00+00:00",
+        "privacy": "public"
+    },
+    "has-session-speakers": true,
```
Comment (Shailesh351) [inline sample/NextcloudConference2017/event]: `is-sessions-speakers-enabled`

File: sample/NextcloudConference2017/tracks [hunk 4]
```diff
@@ -0,0 +1,270 @@
...
+                "id": 3776,
+                "title": "Coding sprint"
+            },
+            {
+                "id": 3775,
+                "title": "Coding sprint"
+            }
+        ],
+        "color": "#C62828",
+        "track_image_url": null,
```
Comment (Shailesh351) [inline sample/NextcloudConference2017/tracks]: Remove this not exists.

File: sample/NextcloudConference2017/sessions [hunk 5]
```diff
@@ -0,0 +1,3197 @@
...
+        "session-type": {
+            "length": "00:05",
+            "id": 328,
+            "name": "Lightning talks"
+        },
+        "starts-at": "2017-08-22T00:00:00+00:00",
+        "audio-url": null,
+        "track": {
+            "color": "#EF6C00",
+            "font_color": "#000000",
```
Comment (Shailesh351) [inline sample/NextcloudConference2017/sessions]: `font-color`

File: sample/NextcloudConference2017/sessions [hunk 6]
```diff
@@ -0,0 +1,3197 @@
...
+            {
+                "city": null,
+                "short-biography": "",
+                "name": "Andy Scherzinger",
+                "speaking-experience": "",
+                "photo-url": "https://storage.googleapis.com/eventyay.com/events/181/speakers/2620/photo/QzlWckp5NX/98e8f859-2d5e-4914-86df-36cdc9071537.png",
+                "organisation": "Contributor",
+                "thumbnail-image-url": "https://storage.googleapis.com/eventyay.com/events/181/speakers/2620/thumbnail/MHh6ZXhHQT/5d1b3fb3-55af-4bd1-93fd-4bbf5f871add.jpg",
+                "long-biography": "",
+                "heard_from": null,
```
Comment (Shailesh351) [inline sample/NextcloudConference2017/sessions]: `heard-from`

File: sample/NextcloudConference2017/speakers [hunk 7]
```diff
@@ -0,0 +1,1402 @@
...
+        "icon-image-url": "",
+        "facebook": null,
+        "github": null,
+        "name": "Simon Peter",
+        "gender": null,
+        "thumbnail-image-url": "",
+        "short-biography": "<p>Simon Peter is the founder and lead developer of the AppImage project, \nand has been a desktop Linux user since over a decade. He thinks that \ndownloading and using applications on the Linux desktop should be at \nleast as hassle-free as on other desktop operating systems.\n</p>",
+        "long-biography": "",
+        "country": "",
+        "small": "",
```
Comment (Shailesh351) [inline sample/NextcloudConference2017/speakers]: `small-image-url`

**Comment by Princu7** [comment]
@Shailesh351 Thanks for reviewing. Made the changes. Please review again :)

**Comment by Princu7** [comment]
@mariobehling Please merge :) Thanks!

**Pull request merged**